app/assets/stylesheets/hammy.css

```diff
@@ -7,9 +7,20 @@
 
 /* Default browser style adds a max width that is hard to override with tailwind directly */
 dialog:modal {
+  background: none;
   max-width: 100vw;
 }
 
+dialog::backdrop {
+  background-color: rgb(255, 255, 255, 0.8);
+}
+
+@media (prefers-color-scheme: dark) {
+  dialog::backdrop {
+    background-color: rgb(0, 0, 0, 0.8);
```

Comment:
@duckinator called out that the modal has the class `bg-opacity-20`, which I take to suggest the intention was an opacity of 20%. I updated these values locally to use `0.2` (20%) and it looks like 👇 What do you think of aligning with that class?

```suggestion
  background-color: rgb(255, 255, 255, 0.2);
}

@media (prefers-color-scheme: dark) {
  dialog::backdrop {
    background-color: rgb(0, 0, 0, 0.2);
```

<img width="1302" height="964" alt="image" src="https://github.com/user-attachments/assets/848887ae-a668-4349-a36f-0ff175fe0580" />

Reply:
That sounds reasonable to me.